### PR TITLE
Adding basic tests for AccessibleObjects

### PR DESCRIPTION
**Pull request by M-Lipin**
Adding basic tests for ComboBoxAccessibleObject, DataGridViewAccessibleObject, PropertyGridAccessibleObject and ToolStripAccessibleObject.

### CODE REVIEW

**Comment by M-Lipin** [comment]
Was not sure about separating DataGridViewAccessibleObject tests in a file as corresponding derived AccessibleObject class is protected relatively to DataGridView.

File: src/System.Windows.Forms/tests/UnitTests/PropertyGridAccessibleObject.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+using Xunit;
+
+namespace System.Windows.Forms.Tests
+{
+    public class PropertyGridAccessibleObjectTests
+    {
+        [Fact]
+        public void PropertyGridAccessibleObject_Ctor_Default()
+        {
+            const string propertyGridAccessibleName = "Properties";
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/tests/UnitTests/PropertyGridAccessibleObject.cs]: @zsd4yr  - Do you know if these tests run only on an ENU locale? Mikhail, I would not risk it, accessible name is localizable. Either replace this test with a null test for the name or set locale for this test.

File: src/System.Windows.Forms/tests/UnitTests/ToolStripAccessibleObject.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            ToolStrip toolStrip = new ToolStrip();
+            yield return new object[] { toolStrip.AccessibilityObject };
+        }
+
+        [Theory]
+        [MemberData(nameof(ToolStripAccessibleObject_TestData))]
+        public void ToolStripAccessibleObject_FragmentNavigate_FirstChild_ThumbButton(AccessibleObject accessibleObject)
+        {
+            const int UIA_ControlTypePropertyId = 30003;
+            const int UIA_ThumbControlTypeId = 50027;
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/tests/UnitTests/ToolStripAccessibleObject.cs]: Consider moving these consts into a common file, it will be useful for other accessibility unit tests.

**Comment by Tanya-Solyanik** [comment]
Looks good, I think it's good to separate Accessible object tests into new files because DGV deserves a lot of tests, and this way we'll have more manageable files. I would move all accessibleobject tests into a new subfolder under tests/unitests. And common native constants into https://github.com/dotnet/winforms/tree/master/src/Common/tests/InternalUtilitiesForTests/src

File: src/System.Windows.Forms/tests/UnitTests/PropertyGridAccessibleObject.cs (hunk 1 above)
Comment (RussKie) [reply src/System.Windows.Forms/tests/UnitTests/PropertyGridAccessibleObject.cs]: `[UseCulture("en-US")]`
https://stackoverflow.com/a/42880541/2338036

Comment (zsd4yr) [reply src/System.Windows.Forms/tests/UnitTests/PropertyGridAccessibleObject.cs]: @Tanya-Solyanik I do not know that. I would not assume anything

@RussKie if that is a valid option, then @M-Lipin it may be prudent to write **at least one** test for all such locales that we support, and then do the remaining tests in a single locale.

**Comment by zsd4yr** [comment]
> Looks good, I think it's good to separate Accessible object tests into new files because DGV deserves a lot of tests, and this way we'll have more manageable files. I would move all accessibleobject tests into a new subfolder under tests/unitests. And common native constants into https://github.com/dotnet/winforms/tree/master/src/Common/tests/InternalUtilitiesForTests/src

I think this is a fine idea!

File: src/System.Windows.Forms/tests/UnitTests/PropertyGridAccessibleObject.cs (hunk 1 above)
Comment (M-Lipin) [reply src/System.Windows.Forms/tests/UnitTests/PropertyGridAccessibleObject.cs]: Thanks for the comments. For this particular test I'll make it locale-independent. But it would be cool to add some localization tests in future PRs.

@RussKie the `UseCultureAttribute` is in the [samples.xunit](https://github.com/xunit/samples.xunit) project and as I see cannot be referenced at this time. To reduce the number of changes in the current PR, did not add additional references.

File: src/System.Windows.Forms/tests/UnitTests/ToolStripAccessibleObject.cs (hunk 2 above)
Comment (M-Lipin) [reply src/System.Windows.Forms/tests/UnitTests/ToolStripAccessibleObject.cs]: I'll do. Thanks!

File: src/Common/tests/InternalUtilitiesForTests/src/NativeConstants.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    public static class NativeConstants
+    {
+        public const int UIA_ControlTypePropertyId = 30003;
+
```
Comment (Tanya-Solyanik) [inline src/Common/tests/InternalUtilitiesForTests/src/NativeConstants.cs]: extra line

File: src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridAccessibleObject.cs [hunk 4]
```diff
@@ -3,6 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 using Xunit;
+using Xunit.Abstractions;
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridAccessibleObject.cs]: What is Abstractions namespace used for in this file?

File: src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridAccessibleObject.cs [hunk 5]
```diff
@@ -11,14 +12,11 @@ public class PropertyGridAccessibleObjectTests
         [Fact]
         public void PropertyGridAccessibleObject_Ctor_Default()
         {
-            const string propertyGridAccessibleName = "Properties";
-
             PropertyGrid propertyGrid = new PropertyGrid();
-            propertyGrid.AccessibleName = propertyGridAccessibleName;
 
             var accessibleObject = new PropertyGridAccessibleObject(propertyGrid);
             Assert.NotNull(accessibleObject.Owner);
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridAccessibleObject.cs]: Ah, I misread your test originally! I missed line 17, where you set the `AccessibleName` property on the owner class and then read it from the AccessibleObject :). So your test had never been locale-specific. Sorry about this. Feel free to add it back, but as a new `Fact` because it tests how the Control.AccessibleName and AccessibleObject.Name propertiea work

File: src/System.Windows.Forms/tests/UnitTests/PropertyGridAccessibleObject.cs (hunk 1 above)
Comment (RussKie) [reply src/System.Windows.Forms/tests/UnitTests/PropertyGridAccessibleObject.cs]: Indeed it is, I should have checked it myself first. We do culture-sensitive tests with NUnit, and I just extended the concept to xUnit...
It looks like an oversight on behalf of the xUnit's team, every repo on GH is copying xUnit's implementation for that attribute.

File: src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridAccessibleObject.cs (hunk 4 above)
Comment (zsd4yr) [reply src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridAccessibleObject.cs]: Looks like nothing

File: src/System.Windows.Forms/tests/UnitTests/PropertyGridAccessibleObject.cs (hunk 1 above)
Comment (RussKie) [reply src/System.Windows.Forms/tests/UnitTests/PropertyGridAccessibleObject.cs]: Digging further, `UseCultureAttribute` doesn't seem to be meant for consumers - https://github.com/xunit/xunit/issues/290.
However there are [culture-related attributes: `[CulturedFact]` and `[CulturedTheory]` in test.utility project](https://github.com/xunit/xunit/issues/1605#issuecomment-355439998).

Worth noting, there may be edge-cases to be aware of - https://github.com/xunit/xunit/issues/1605

File: src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridAccessibleObject.cs (hunk 5 above)
Comment (M-Lipin) [reply src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridAccessibleObject.cs]: I see, thank you for this and previous comments! I'll add this later, maybe the locale-specific fact(s) would be better for the accessible name test.

File: src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridAccessibleObject.cs (hunk 4 above)
Comment (M-Lipin) [reply src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridAccessibleObject.cs]: Thanks, I'll remove this.
